litevideo/csc/common.py

```diff
@@ -18,13 +18,21 @@ def saturate(i, o, minimum, maximum):
 def coef(value, cw=None):
     return int(value * 2**cw) if cw is not None else value
 
-
 def rgb_layout(dw):
     return [("r", dw), ("g", dw), ("b", dw)]
 
+def rgb16f_layout(dw):
+    return [("r_f", dw), ("g_f", dw), ("b_f", dw)]
 
 def ycbcr444_layout(dw):
     return [("y", dw), ("cb", dw), ("cr", dw)]
 
 def ycbcr422_layout(dw):
     return [("y", dw), ("cb_cr", dw)]
+
+def pix_layout(dw):
+    return [("pix", dw)]
+
+def pixf_layout(dw):
+    return [("pixf", dw)]
```

Comment:
Replaced "r_f" with "rf", retained everything else as it makes it easier to understand the code.